SpatialGDK/Source/SpatialGDK/Private/Utils/RPCContainer.cpp

```diff
@@ -126,10 +126,18 @@ void FRPCContainer::ProcessRPCs(FArrayOfParams& RPCList)
 
 void FRPCContainer::ProcessRPCs()
 {
+	if (bAlreadyProcessingRPCs)
+	{
+		UE_LOG(LogRPCContainer, Warning, TEXT("Calling ProcessRPCs recursively, ignoring the call"));
```

Comment:
I think because this can happen naturally, and *shouldn't* have any adverse reaction, this could be a Log

Reply:
Updated.